CHANGELOG.md

```diff
@@ -21,12 +21,16 @@ You can participate in the conversation around this release in [this issue](ADD)
 - Remove TextInput's `inputView` prop ([1804e7cbea](https://github.com/facebook/react-native/commit/1804e7cbea707a35046118090966a54031edfae8) by [@TheSavior](https://github.com/TheSavior))
 - Animated: Remove `defaultProps` Parameter ([a70987cee2](https://github.com/facebook/react-native/commit/a70987cee24bcd027b9c4a5aa85dfd6a1aab74b3) by [@yungsters](https://github.com/yungsters))
 - RN: Delete `TextInput.selectionState` Prop ([2becdfd404](https://github.com/facebook/react-native/commit/2becdfd4041f7f28138ba3a61c03e17c06dc2e50) by [@yungsters](https://github.com/yungsters))
+- Remove accessibilityStates property. ([7b35f427fd](https://github.com/facebook/react-native/commit/7b35f427fd66cb0f36921b992095fe5b3c14d8b9))
+- Remove framesToPop from bridge ([a483f802fd](https://github.com/facebook/react-native/commit/a483f802fddfd927f2baa0d95e2b4094d452cddd) by [@motiz88](https://github.com/motiz88))
```

Comment:
Is this not considered internal?

Reply:
I think framesToPop will have some kind of external implications, let's leave this with a TODO to figure out what the change is and how to message it in the changelog

Reply:
cc @motiz88

Reply:
@rickhanlonii This commit isn't a breaking change. I believe you mean https://github.com/facebook/react-native/commit/8bc02fdd52124d0a24d96e4a61d7688328ef1660 (which is in the changelog btw), but that's not breaking either because React Native ships with an alternative mechanism already enabled and satisfactory defaults included for both existing and new apps.

I don't believe `framesToPop` was ever a documented API, more of a Facebookism that spread to some of our packages (like `invariant`). Looking at de facto external implications: Crash reporting services might see some extra stack frames we would previously filter out. That's _if_ they weren't already doing their own thing with a custom `ExceptionsManager` (IIRC Sentry was the one I've seen do this).